### PR TITLE
Update UI plugins on Astro note

### PR DESCRIPTION
**Pull request by TJaniF**
Moved the note to the part about Appbuilder views, since it is more relevant there (external links work on Astro the same way as on local Airflow without needing these permissions, just tested that :) )

Also added the code parts into the main example for the people just copy-pasting and commented out the ones that would potentially break it locally / in other Airflow setups. 

Tested the plugin locally and on Astro :) (really cool that you can do that now!)

### CODE REVIEW

**Review by kentdanas — APPROVED** [review]

File: learn/using-airflow-plugins.md [hunk 1]
```diff
@@ -191,6 +148,9 @@ You can add a view to render a simple templated HTML file on top of the Airflow
     from airflow.plugins_manager import AirflowPlugin
     from flask import Blueprint
     from flask_appbuilder import expose, BaseView as AppBuilderBaseView
+    from airflow.configuration import conf
+    # from airflow.www.auth import has_access  # uncomment for Astro
+    # from airflow.security import permissions  # uncomment for Astro
```
Comment (jwitz) [inline learn/using-airflow-plugins.md:153]:
```suggestion
    # from airflow.www.auth import has_access  # uncomment to use the plugin on Astro
    # from airflow.security import permissions  # uncomment to use the plugin on Astro
```

File: learn/using-airflow-plugins.md [hunk 2]
```diff
@@ -205,20 +165,25 @@ You can add a view to render a simple templated HTML file on top of the Airflow
         default_view = "test"
 
         @expose("/")
+        # @has_access([(permissions.ACTION_CAN_ACCESS_MENU, "Custom Menu")]) # uncomment for Astro
```
Comment (jwitz) [inline learn/using-airflow-plugins.md:168]:
```suggestion
        # @has_access([(permissions.ACTION_CAN_ACCESS_MENU, "Custom Menu")]) # uncomment to use the plugin on Astro
```

File: learn/using-airflow-plugins.md [hunk 3]
```diff
@@ -240,6 +205,53 @@ By clicking on **Test View** you can access the Flask View that was defined as `
 
 ![Test View](/img/guides/plugins_test_view.png)
 
+:::info
+
+If you want to use custom menu items in an Airflow environment hosted on Astro, you must make sure to give your plugin the necessary permissions. To do this, use the `@has_access` decorator to give your BaseView method `ACTION_CAN_ACCESS_MENU` permissions.
+
+    ```python
+    from airflow.www.auth import has_access  # uncomment for Astro
+    from airflow.security import permissions  # uncomment for Astro
```
Comment (jwitz) [inline learn/using-airflow-plugins.md:214]:
```suggestion
    from airflow.www.auth import has_access
    from airflow.security import permissions 
```

File: learn/using-airflow-plugins.md [hunk 4]
```diff
@@ -240,6 +205,53 @@ By clicking on **Test View** you can access the Flask View that was defined as `
...
+
+    # ...
+
+    # create a flask appbuilder BaseView
+    class MyBaseView(AppBuilderBaseView):
+        default_view = "test"
+
+        @expose("/")
+        @has_access([(permissions.ACTION_CAN_ACCESS_MENU, "Custom Menu")]) # uncomment for Astro
```
Comment (jwitz) [inline learn/using-airflow-plugins.md:223]:
```suggestion
        @has_access([(permissions.ACTION_CAN_ACCESS_MENU, "Custom Menu")]) 
```

**Review by jwitz — APPROVED** [review]
Brilliant! Thanks so much for cleaning up my rush work  @TJaniF

**Pull request merged**